app/views/visitors/index.html.erb

```diff
@@ -1,24 +1,86 @@
-<!-- TODO: Remove template home page and any assets -->
-<section class="container">
-  <div class="row">
-    <aside class="col">
-      <%= image_tag("dxw-logo-512x512.png") %>
-    </aside>
-    <article class="col">
-      <h1>Rails Template</h1>
-      <p>Welcome to dxw's template. This application contains our latest defaults for use when starting a new repository to get going as quickly as possible.</p>
-      <p>On first use please refer to the template <%= link_to "README", "https://github.com/dxw/dfsseta-apply-for-landing-ruby", target: :blank %> for further instructions.
-      <p>This home page should be removed during initial set up.</p>
+<main class="govuk-main-wrapper">
+  <section class="container">
+    <div class="row">
+      <article class="col">
+        <h1 class="govuk-heading-xl">Apply for landing</h1>
 
-      <h2>Helpful links</h2>
-      <ul>
-        <li>
-          <%= link_to "Architecture decision records", "https://github.com/dxw/dfsseta-apply-for-landing-ruby/tree/develop/doc/architecture/decisions" %>
-        </li>
-        <li>
-          <%= link_to "dxw RFCs", "https://github.com/dxw/tech-team-rfcs/" %>
-        </li>
-      </ul>
-    </article>
-  </div>
-</section>
+        <p class="govuk-body">Welcome to the service</p>
+        <p class="govuk-body">Refer to our <%= link_to "README", "https://github.com/dxw/dfsseta-apply-for-landing-ruby", class: "govuk-link", target: :blank %> for further instructions.
+
+        <h2 class="govuk-heading-l">GOV UK Design System "accordion" test</h2>
+          <div class="govuk-accordion" data-module="govuk-accordion" id="accordion-default">
+            <div class="govuk-accordion__section">
+              <div class="govuk-accordion__section-header">
+                <h2 class="govuk-accordion__section-heading">
+                  <span class="govuk-accordion__section-button" id="accordion-default-heading-1">
+                    Overview
+                  </span>
+                </h2>
+              </div>
+              <div id="accordion-default-content-1" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-1">
```

Comment:
Order of `id` and `class` is changing here

Reply:
hmm, not sure what you're saying here about ordering of `id` and `class` attributes?  I copied the accordion markup from https://design-system.service.gov.uk/components/accordion/ Could you please elaborate?

Reply:
Just that previous tags put `class` before `id`, but this and one other flip it. Very minor, but it might be nice to keep it consistent at least within the same file. I can see that the example doesn't do that though!